GoogleForCreators/web-stories-wp
# PPCv2.0 Help Center + Checklist Toggles

## Context
Adding in the new checklist popup and this happens:
<img width="541" alt="Screen Shot 2021-06-25 at 2 16 03 PM" src="https://user-images.githubusercontent.com/10720454/123486230-7b0fc400-d5c0-11eb-8fb8-58fd98185ac8.png">


## Summary

Sam was kind enough to help me figure out a best path forward on a friday for this. 
If you want to read our chat --> https://xwp.slack.com/archives/C01ESBB8ZN1/p1624570880176300 

She suggested this implementation of the small screen toggles: https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=8550%3A61310 
<img width="364" alt="Screen Shot 2021-06-25 at 2 22 54 PM" src="https://user-images.githubusercontent.com/10720454/123486427-d80b7a00-d5c0-11eb-954c-bdc8ed654932.png">

## Relevant Technical Choices

- Creating a shared `ToggleButton` resource for both the help center and checklist to cut down on some duplicate code. For now it's in its own directory just within `components`. This PR has to go through QA and we have other work to do on this refactor. I didn't want to mess with excess path updates until I have to. I also don't think this belongs in the design system since it's pretty specific to these two popups.

## To-do

Sam agreed that for now we can use the small screen version of these toggle buttons but that we should eventually make sure this whole carousel area is responsive. Since that's just a litttttle bit out of scope on updating the checklist, I've created a ticket for that here: #8086 

## User-facing changes

Help Center button has changed, now it just has a question mark icon and the number of unread notifications. If there's no unread notifications then just the icon shows. No more caret. It also has a tooltip now. 

![Screen Shot 2021-06-25 at 3 42 13 PM](https://user-images.githubusercontent.com/10720454/123491685-2bcf9080-d5cc-11eb-9382-4ccf5d2a0e4a.png)

![Kapture 2021-06-25 at 15 44 28](https://user-images.githubusercontent.com/10720454/123491720-41dd5100-d5cc-11eb-9a50-de80d7a10d1b.gif)



## Testing Instructions

Verify the above and make sure help center works as expected.

### QA

Verify these user facing changes and make sure the help center works as expected (Checklist is behind a feature flag so you won't see that yet).




This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #7917
